scripts/clean_mozart_indexes.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 ES_URL=$1
 
-curl -XDELETE "${ES_URL}/job_status-current"
```

Comment:
i think we can simplify this:
```bash
$ curl -XDELETE "${ES_URL}/job_status-*"
```

https://www.elastic.co/guide/en/elasticsearch/reference/7.10/indices-delete-index.html
> (Request, string) Comma-separated list or wildcard expression of indices to delete.

Reply:
@DustinKLo I initially tried your suggestion but was getting an error about wildcards not being allowed:

```
(mozart) hysdsops@ip-100-104-10-166:~/mozart/ops/nisar-pcm/cluster_provisioning$ curl -XDELETE "http://localhost:9200/job_status-*"
{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"Wildcard expressions or all indices are not allowed"}],"type":"illegal_argument_exception","reason":"Wildcard expressions or all indices are not allowed"},"status":400}
(mozart) hysdsops@ip-100-104-10-166:~/mozart/ops/nisar-pcm/cluster_provisioning$
``` 

Can you try this on your local cluster?

Reply:
you're correct, looks like we need to enable certain settings to allow for this:
https://www.elastic.co/guide/en/elasticsearch/reference/7.10/indices-delete-index.html#delete-index-api-path-params

> To delete all indices, use _all or * . To disallow the deletion of indices with _all or wildcard expressions, change the action.destructive_requires_name cluster setting to true. You can update this setting in the elasticsearch.yml file or using the [cluster update settings](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cluster-update-settings.html) API.

Reply:
What's weird is that in the existing scripts, there are command line calls to deleting with wildcards, which doesn't make any sense:

https://github.com/hysds/hysds/blob/e964f45ddf96fab402a14a5f63aa534a3b650d0a/scripts/clean_mozart_indexes.sh#L8

Reply:
looks like it was committed 6 years ago, when HySDS was still using Elasticsearch 1.7